template/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 # include some defines automatically made by qpm
 include(qpm_defines.cmake)
 
+# override mod id
+set(MOD_ID "#{id}")
```

Comment:
Just to explain this in more detail, by default `qpm-rust` will set the id to be the mod name, just with all the spaces removed. This generally works, but it's better to explicitly state the mod id.

For example, in Hot-Swappable Mods, I set the id to be `HotSwappableMods`, yet the mod was built with the id `Hot-SwappableMods`. Obviously it's not the end of the world, but there's no harm in explicitly stating the mod id